exercises/gigasecond/description.md

```diff
@@ -1,3 +1,4 @@
-Calculate the moment when someone has lived for 10^9 seconds.
+Given a `start` value, write a function that returns what the date will
```

Comment:
This is a good point.  It should be obvious from the test suite what form the expected result should take.  Obviously the student must figure out how to get there.  By stating `date` here it also locks all tracks into a style of expected output.  I don't know what other form the output might take, but by being, perhaps, too explicit we are driving the student to a single solution, as you stated.  With this new insight my approval is wavering....

Reply:
I would say "a form of the expected output" since I believe that the tests assume that given the result we can form a date.  However, given a moment as "some amount of seconds as expressed as an integer" we should be able to expect that second given increased by a gigasecond worth of seconds.  That is the fundamental and basic thing that I expect out of the method.  The tests can use that to form whatever form of that as a test that it wants to.  I say "give the most basic thing back as is possible" in this case.  

Perhaps the change should be that there is a test that affirms that it is at the core, like the name "Gigasecond" implies a representation of "seconds" in its most basic form.

Reply:
Of course, I am speaking about this from the Ruby perspective.  This is already done if I remember correctly on the Bash track and I would have to double check on the Vim track.